conwayste/src/client.rs

```diff
@@ -522,15 +389,14 @@ impl MainState {
             video_settings:      vs,
             config:              config,
             viewport:            viewport,
-            input_manager:       input::InputManager::new(input::InputDeviceType::PRIMARY),
+            inputs:              input::InputManager::new(),
             net_worker:          None,
             single_step:         false,
             arrow_input:         (0, 0),
             drag_draw:           None,
-            win_resize:          0,
-            return_key_pressed:  false,
-            escape_key_pressed:  false,
             toggle_paused_game:  false,
+            power_on_timestamp:  0.0,
```

Comment:
Can we name this something else? It's not really "powering on" since it's an app.

Reply:
Sure. Bad habits die hard lol